LumpNav/LumpNav.py

```diff
@@ -40,18 +40,71 @@ def __init__(self, parent = None):
     GuideletWidget.__init__(self, parent)
     
   def setup(self):
+    # Adds default configurations to Slicer.ini
+    self.addDefaultConfiguration()
+    
     GuideletWidget.setup(self)
-
+    
   def addLauncherWidgets(self):
     GuideletWidget.addLauncherWidgets(self)
 
+    # Configurations
+    self.configurations()
+    
+    # BreachWarning
+    self.breachWarningLight()  
+ 
+  # Adds default configurations to Slicer.ini
+  def addDefaultConfiguration(self):
+    settings = slicer.app.userSettings()    
+    if not settings.value(self.moduleName + '/Configurations/default/TipToSurfaceDistanceCrossHair'):

```

Comment:
Yes, I kind of suspected this wasn't a very pretty solution.

About contains(), I don't think it works if it is a 'sub-key', e.g:

>>> settings.contains('LumpNav/Configurations/default')
False

while:

>>> settings.contains('LumpNav/Configurations/default/needleModelToNeedleTip')
True

That is why I did what I did. Do you have anyother suggestions perhaps? I guess I can always create a function that searches at settings.allKeys() for example.